js/label/shared/Label.svelte

```diff
@@ -35,27 +42,35 @@
 			>
 				<div class="inner-wrap">
 					<meter
-						aria-labelledby="meter-text"
+						aria-labelledby={get_aria_referenceable_id(
+							`meter-text-${confidence_set.label}`
+						)}
+						aria-label={confidence_set.label}
+						aria-valuenow={Math.round(confidence_set.confidence * 100)}
+						aria-valuemin="0"
+						aria-valuemax="100"
 						class="bar"
 						min="0"
-						max="100"
+						max="1"
+						value={confidence_set.confidence}
 						style="width: {confidence_set.confidence *
 							100}%; background: var(--stat-background-fill);
 						"
-						value="100"
-						aria-label={Math.round(confidence_set.confidence * 100) + "%"}
 					/>
 
 					<dl class="label">
-						<dt id={`meter-text-${confidence_set.label}`} class="text">
+						<dt
+							id={get_aria_referenceable_id(
+								`meter-text-${confidence_set.label}`
+							)}
+							class="text"
+						>
 							{confidence_set.label}
 						</dt>
-						{#if value.confidences}
```

Comment:
`if value.confidences` is already check by the outer if-block, so this can be removed?